plugin_tests/cli_results_test.py

```diff
@@ -44,15 +45,15 @@ def tearDownModule():
 
 
 class CliResultsTest(unittest.TestCase):
-    def _runTest(self, cli_args=(), cli_kwargs={}, outputs={}, contains=[],
-                 excludes=[]):
-        """
-        Test a cli by calling subprocess.  Ensure that output files match a
-        sha256 and stdout contains certain phrases and excludes other phrases.
-        A value in the cli_args or cli_kwargs of 'tmp_<value>' will create a
-        temporary file and use that instead.  The same value in the output
-        list can be used to test the existence and contents of that file at the
-        end of the process.
+    def _runTest(self, cli_args=(), cli_kwargs={}, outputs={},
+                 in_process=True, contains=[], excludes=[]):
+        """Test a cli by calling runpy or subprocess.  Ensure that output
```

Comment:
I can do that to match within the file. More generally, we seem to be pretty split -- by my count, 68 of the other 112 doc strings in the entire project start on the same line, and the others don't.